pennylane/operation.py

```diff
@@ -1205,23 +1206,51 @@ def expand(self):
 
         return tape
 
+    @property
+    def arithmetic_depth(self) -> int:
+        """Arithmetic depth of the operator."""
+        return 0
+
+    def simplify(self) -> "Operator":  # pylint: disable=unused-argument
+        """Reduces to the maximum the depth of nested operators.
```

Comment:
Still a bit unclear what this method is doing, does it simplify to `arithmetic_depth` or, does it maximally simplify? 
```suggestion
        """Reduces to the maximum depth of nested operators.
```

Reply:
It maximally simplifies. I will change the docstring to make it clearer.

Reply:
Hopefully it is clearer now.